event-publisher-spi/src/main/scala/uk/gov/nationalarchives/eventpublisherspi/UserMonitoringTask.scala

```diff
@@ -13,7 +13,7 @@ import io.circe.syntax.EncoderOps
 import java.net.URI
 import scala.jdk.CollectionConverters._
 
-class UserMonitoringTask(snsUtils: SNSUtils, config: EventPublisherConfig, credentialType: String) extends ScheduledTask {
+class UserMonitoringTask(snsUtils: SNSUtils, config: EventPublisherConfig, credentialTypes: List[String]) extends ScheduledTask {
```

Comment:
Maybe change the name of the `credentialTypes` to: `validConfiguredCredentialTypes`? Just to make it a bit clearer what these are?

Reply:
That makes sense. I've changed it.